tests/unit/test_search.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date, datetime, time, timedelta, timezone
+
+import pytest
+from dateutil.parser import ParserError
+from earthaccess import search
+from numpy import datetime64
+from pandas import Timestamp
```

Comment:
Admittedly, versions and environments are kinda a mess right now -- I'll work on that soon

Reply:
N/A now, but I was confused about this because I saw xarray in the dev group and numpy is an xarray dependency. I assumed that CI testing used the dev group, but I don't know. Sounds like that all has an overhaul in its future.